src/quo2/components/wallet/account_card/style.cljs

```diff
@@ -3,32 +3,37 @@
     [quo2.foundations.colors :as colors]))
 
 (defn text-color
-  [watch-only? theme]
-  (if (and watch-only? (= :light theme))
+  [type theme]
+  (if (and (or (= :missing-keypair type)
+               (= :watch-only type))
+           (= :light theme))
     colors/neutral-100
     colors/white))
 
 (defn card
-  [{:keys [customization-color watch-only? metrics? theme pressed?]}]
+  [{:keys [customization-color type theme pressed?]}]
   {:width              162
-   :height             (if metrics? 88 68)
-   :background-color   (if watch-only?
-                         (colors/theme-colors colors/neutral-80-opa-5 colors/neutral-95 theme)
+   :background-color   (when (not= :watch-only type)
```

Comment:
@mmilad75 can this be handled with the custom-color overlay? we used it before on some component we worked on together

Reply:
The `customization-colors/overlay` component is already used in the component but serving a different purpose.

Reply:
yes but I think it still needs to get used for this use case here, pretty sure it's the same concept just using opacity which should get passed to the overlay in these cases

Reply:
The overlays are different in different conditions, that's why I used a custom overlay for this component. I don't think it can be done by the `custom-overlay`